extension-ironsource/ext.manifest

```diff
@@ -1,8 +1,11 @@
 name: "IronSourceExt"
 
 platforms:
+    android:
+        context:
+            excludeJars: ["(.*)/androidx-multidex.jar"]
```

Comment:
Workaround for https://github.com/defold/extender/issues/385 (if libd has dependency on androidx)